src/server.py

```diff
@@ -1,11 +1,15 @@
 # -*- encoding: utf-8 *-*
 # This is an http-server#
-
+from __future__ import unicode_literals
 import socket
-
+import io
 import email.utils
+from mimetypes import guess_type
+import os
+
 
 CRLF = '\r\n'
+ABS_PATH = os.path.abspath(__file__).rsplit('/', 2)[0] + '/webroot'
```

Comment:
look further into your `os.path` library and you'll find `os.path.dirname`.  It can do what you want to do here in a way that works across platforms.  The problem with what you have here is that it wouldn't work on a Windows system.